doc/specifications/proposed/translations/removal.md

```diff
@@ -0,0 +1,55 @@
+# Removal of Content Object Translations
+
+## Deleting single translation from a content object
+
+Sometimes there is a need to delete single translation of a content object without deleting
+the object itself.
+
+To aid that, PHP API introduces the `deleteTranslation` method on `ContentService` specified as:
+
+```php
+/**
+ * Delete content object translation by creating new version draft without the specified translation.
+ *
+ * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException if the specified translation
+ *         is the only one content object has.
+ *
```

Comment:
you should add on both:
`* @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException if the user is not allowed to delete the content (in one of the locations of the given content object)`